src/scrabble.js

```diff
@@ -1,5 +1,31 @@
-function scrabble() {
-  // write code here
+function scrabble(scrabbleword) {
+
+  let word = scrabbleword
+  
+  if (word === null) {
```

Comment:
im not 100% with what i'm about to say, but could you put the if statement as one or does it have to be two different one's?

Reply:
I would second Callum on this part, both `if` statements are returning 0 so you could potentially combine them into a single line of code using another `||` comparison